release-notes.md

```diff
@@ -282,6 +287,50 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v33.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v33.0. The following user visible changes have been made:
+
+### D-Bus based API
+
+A D-Bus based API is added as an alternative to the existing REST
+API. This feature is gated by the `dbus_api` feature. Details can be
+found from the [API documentation](docs/api.md).
```

Comment:
s/found from/found in/

Reply:
Thanks Rob for the detailed review. It has always been challenging for me to use the right prepositions (in, on, from, of, etc.). Do you happen to have good references about how to properly use prepositions?